tests/Resources/Page/OCPDashboard/InstalledOperators/InstalledOperators.robot

```diff
@@ -40,6 +40,7 @@ Search Installed Operator
 
 Is Operator Installed
   [Arguments]  ${operator}
+  Run Keyword and Return Status     Wait Until Element is Visible      //a[@data-test-operator-row="${operator}"]    timeout=10
```

Comment:
I found this redundant with the line 44. You could use 43 to get the status since the WebElement is not returned

Reply:
i know it's redundant i wanted to make sure web element is present. i just added it to remove anonymous issue like iif selenium somehow able to found element visible. If you agree I think we could handle this in refactoring or if you want i can push the change.

Reply:
as you prefer

Reply:
Sure..I am resolving this conversation